src/gatsby-node.js

```diff
@@ -120,7 +120,7 @@ exports.sourceNodes = async (
   { actions, store, cache, createNodeId },
   options
 ) => {
-  const { createNode, touchNode } = actions
+  const { createNode, touchNode, getNode } = actions
```

Comment:
I don't think this works. At least it didn't work for me on gatsby v4. I got getNode like this:
```javascript
exports.sourceNodes = async (
  { actions, store, cache, createNodeId, getNode },
  options
) => {
```

Reply:
You are correct. This change is required.